install-ci.yml

```diff
@@ -14,11 +14,19 @@
       tags:
         - monitoring
 
-    - role: letsencrypt
+    - role: letsencrypt-apache
       tags:
         - letsencrypt
       when:
         - letsencrypt_csr_cn | default(False)
+        - bonnyci_use_apache
```

Comment:
I'm a little concerned that these conditional items aren't piped through the bool filter, but not enough to block this.